### PR TITLE
dev-util/kcov: add 43 and adoption

### PR DESCRIPTION
**Pull request by beatussum**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-util/kcov/kcov-43.ebuild [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9,10} )
```
Comment (juippis) [inline dev-util/kcov/kcov-43.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{10..13} )
```

File: dev-util/kcov/kcov-43.ebuild [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	dev-libs/elfutils
+	dev-libs/libunistring:=
+	dev-libs/openssl:=
+	net-dns/c-ares:=
+	net-dns/libidn2:=
+	net-libs/libpsl
+	net-misc/curl
+	net-libs/nghttp2:=
+	sys-libs/binutils-libs:=
+	sys-libs/zlib:=
```
Comment (juippis) [inline dev-util/kcov/kcov-43.ebuild:36]:
```suggestion
	sys-libs/zlib
```

**Review by juippis — COMMENTED** [review]
This doesn't apply cleanly to the git tree anymore, could you rebase and I'll take a look asap?

File: dev-util/kcov/kcov-43.ebuild (hunk 1 above)
Comment (beatussum) [reply dev-util/kcov/kcov-43.ebuild]: Normally, it's done!

File: dev-util/kcov/kcov-43.ebuild (hunk 2 above)
Comment (beatussum) [reply dev-util/kcov/kcov-43.ebuild]: I'm not sure to understand why I should remove the `:=`. `sys-libs/zlib` has a subslot. Sorry in advance, maybe I missed something. However, I've resolved that.

**Review by juippis — APPROVED** [review]
Thanks!

File: dev-util/kcov/kcov-43.ebuild (hunk 2 above)
Comment (juippis) [reply dev-util/kcov/kcov-43.ebuild]: https://github.com/ionenwks/iwdevtools/blob/master/config/qa-vdb.exclude-bind#L5

**Pull request closed without merging**